src/hello_client.erl

```diff
@@ -165,12 +166,17 @@ handle_call(terminate, _From, State) ->
 %% @hidden
 handle_info({?INCOMING_MSG, Message}, State) ->
     incoming_message(Message, State);
+handle_info(?PING, State = #client_state{ping_counter = PingCounter}) when PingCounter >= 3 ->
+    ?LOG_ERROR("There is no PONG answer on PING for a long time. Client will be restarted.", []),
```

Comment:
@surik at least the time hello_client was waiting for a $PONG should be logged when the PingCounter is >=3. Assuming the $PING is sent approximately right away from the client the resulting time measurement 3 * keep_alive_interval should be fine. Something like "Ping request to <IP> timed out after 600s. Client will be restarted." is imho ok. Additional meta fields like server IP or the name of the PID (if named) would be very nice but this stuff  is atm also not part of the other logging (see hello_log.hrl). We should fix that in the future.